sites/org/pages/docs/[...slug].mjs

```diff
@@ -1,191 +1,7 @@
-// Used in static paths
-import { mdxPaths } from 'site/prebuild/mdx-paths.en.mjs'
-// Dependencies
+import { DocsPage, ns as pageNs } from 'shared/components/docs/docs-page.mjs'
+import { mdxLoader } from 'shared/mdx/v3loader.mjs'
+import { mdxPaths } from 'site/prebuild/docs/mdx-paths.en.mjs'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
-// Hooks
-import { useState, useEffect } from 'react'
-// Components
-import Head from 'next/head'
-import { PageWrapper, ns } from 'shared/components/wrappers/page.mjs'
-import { Spinner } from 'shared/components/spinner.mjs'
-import { components } from 'shared/components/mdx/index.mjs'
-import { MdxWrapper } from 'shared/components/wrappers/mdx.mjs'
-import { Toc } from 'shared/components/mdx/toc.mjs'
-import { DocsLayout } from 'site/components/layouts/docs.mjs'
-
-/*
- * PLEASE READ THIS BEFORE YOU TRY TO REFACTOR THIS PAGE
```

Comment:
Did you read this?
Because you're reverted the way we load MDX to the v2 way, which means markdown changes are no longer picked up for fast-refresh.

We're essentially back to where we were 3 months ago when it comes to loading MDX.
I don't understand why you would unilaterally implement such a significant change that essentially rolls back a bunch of work that's been done.

Reply:
I did, but I focused more on the don't DRY it up part than the don't load from filesystem part. I went back and forth on dynamic loading vs filesystem loading. All the docs files need to be compiled and bundled anyway because we load them dynamically in the workbench, so there's not a big build time saving from doing it this way instead, but in terms of bundle size and load time, as well as compile time on dev, it's major. 

I wonder if we might consider using a [watch plugin](https://github.com/hashicorp/next-remote-watch) to fix the hot-reload issue.